### PR TITLE
build separated commands into a single command line builder

### PR DESCRIPTION
**Pull request by pkErbynn**
this combines separated modes (tool, luncher) commands into a single consolidated command line builder,
allowing automatic dynamic runtime mode determination and code command reusability

### CODE REVIEW

**Comment by pkErbynn** [comment]
@bagdxk kindly review and give feedback if any, thanks

File: src/Crossroads/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Crossroads": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "inspect --package newnotepad.exe"
```
Comment (bagdxk) [inline src/Crossroads/Properties/launchSettings.json:5]: not add test in config.

**Review by bagdxk — CHANGES_REQUESTED** [review]

File: src/Crossroads/Properties/launchSettings.json (hunk 1 above)
Comment (pkErbynn) [reply src/Crossroads/Properties/launchSettings.json]: alright sure, will remove it

**Comment by pkErbynn** [comment]
closed as currently merged pr has all changes

**Pull request closed without merging**